src/main/resources/changelogs/deleteInitialData.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.0.xsd">
+
+    <changeSet author="radoslawl" id="Issue142-deleteInitialData-slots">
+        <delete tableName="slot"></delete>
+    </changeSet>
+
+    <changeSet author="radoslawl" id="Issue142-deleteInitialData-notes">
+        <delete tableName="note"></delete>
+    </changeSet>
+
+    <changeSet author="radoslawl" id="Issue142-deleteInitialData-candidates">
+        <delete tableName="candidate"></delete>
+    </changeSet>
+
+    <changeSet author="radoslawl" id="Issue142-deleteInitialData-recruiter_note">
+        <delete tableName="recruiter_note"></delete>
+    </changeSet>
+
+    <changeSet author="radoslawl" id="Issue142-deleteInitialData-persons">
+        <preConditions>
+            <changeSetExecuted id="Issue142-deleteInitialData-slots" author="radoslawl" changeLogFile="changelogs/deleteInitialData.xml"></changeSetExecuted>
+            <changeSetExecuted id="Issue142-deleteInitialData-notes" author="radoslawl" changeLogFile="changelogs/deleteInitialData.xml"></changeSetExecuted>
+            <changeSetExecuted id="Issue142-deleteInitialData-candidates" author="radoslawl" changeLogFile="changelogs/deleteInitialData.xml"></changeSetExecuted>
+            <changeSetExecuted id="Issue142-deleteInitialData-recruiter_note" author="radoslawl" changeLogFile="changelogs/deleteInitialData.xml"></changeSetExecuted>
+        </preConditions>
+
+        <delete tableName="person">
+            <where>
+                email!='recruiter@test'
```

Comment:
why leave the one recruiter? Maybe instead just add one recruiter - for Michalina